src/v/config/configuration.h

```diff
@@ -248,14 +248,14 @@ struct convert<std::chrono::milliseconds> {
 
 inline ss::sstring to_absolute(const ss::sstring& path) {
     namespace fs = std::filesystem;
-    return fs::absolute(fs::path(path)).native();
+    return path.empty() ? std::string() : fs::absolute(fs::path(path)).native();
```

Comment:
use boost or absl

Reply:
fixed that
didn't find filesytstem library in absl and ended up using boost
checked that behaviour doesn't change depending on a build